util.js

```diff
@@ -40,3 +40,15 @@ exports.getAvaConfig = filepath => {
 exports.isFunctionExpression = node => {
 	return node && functionExpressions.indexOf(node.type) !== -1;
 };
+
+exports.getTestModifiers = function getTestModifiers(node) {
```

Comment:
Looking forward to it, but no idea how long that'll take before we can put that in modules without transpiling... End of life of Node v6 maybe...? :/